.ci.yaml

```diff
@@ -204,7 +204,6 @@ platform_properties:
           {"dependency": "certs", "version": "version:9563bb"}
         ]
       os: Windows
-      device_type: none
```

Comment:
Is this the right change to make, or is the issue that the bot is missing this dimension?

Reply:
That's a good question. Let me loop back on that.

Reply:
In this case, which is odd, I believe this to be the correct change as the target Windows-11 machines do not have this dimension. So attempting to match on this actually excludes them.

Reply:
Here is an example: https://chromium-swarm.appspot.com/bot?id=flutter-win-10

If you go to the bot list searching on device_type:none it does not appear there: https://chromium-swarm.appspot.com/botlist?c=id&c=task&c=os&c=status&d=asc&f=pool%3Aluci.flutter.staging&f=device_type%3Anone&k=device_type&s=id

Which admittedly I find odd and would have assumed that this was the default.

Reply:
> In this case, which is odd, I believe this to be the correct change as the target Windows-11 machines do not have this dimension. So attempting to match on this actually excludes them.

Right, I was wondering if this was a mistake, and if the true fix should not be adding the dimension to be consistent with the other devicelab bots.

Reply:
> > In this case, which is odd, I believe this to be the correct change as the target Windows-11 machines do not have this dimension. So attempting to match on this actually excludes them.
> 
> Right, I was wondering if this was a mistake, and if the true fix should not be adding the dimension to be consistent with the other devicelab bots.

Ah that is a good point. I am not sure if there is a precedent but you might be right here. It might be more intuitive to have to include this as device_type: none.

Reply:
I talked to @yusuf-goog, the Windows Arm64 devices are physical machines, and normally physical machines have devices attached to them. The Windows x64 devices `device_type: none` are actually VMs. In other words, this is expected from the bots perspective.

Reply:
Ah, I commented below but I will defer to Yusuf. He knows more about this than I do so go for it.

Reply:
Small note: we do have windows x64 devices in the devicelab as well, so builds should target the right dimensions to pick the arm64 win bots.

Reply:
> I talked to @yusuf-goog, the Windows Arm64 devices are physical machines, and normally physical machines have devices attached to them. The Windows x64 devices `device_type: none` are actually VMs. In other words, this is expected from the bots perspective.

ahh yeah, of course!